src/main/java/de/eposcat/master/approachImpl/JSON_Oracle_DatabaseAdapter.java

```diff
@@ -82,6 +91,27 @@ public Page loadPage(int pageId) throws SQLException{
         }
     }
 
+    @Override
+    public List<Page> findPagesByType(String type) throws SQLException {
+        PreparedStatement stFindByType = conn.prepareStatement("SELECT * FROM pages WHERE type = ?");
+        stFindByType.setString(1,type);
+
+        ResultSet rs = stFindByType.executeQuery();
+        List<Page> pages = new ArrayList<>();
+
+        while(rs.next()){
+            long id = rs.getInt("ID");
+            String resultType = rs.getString("type");
+            String attributesJSON = rs.getString("attributes");
+
+            Page page = new Page(id, resultType);
+            page.setAttributes(jsonToMap(attributesJSON));
+            pages.add(page);
+        }
+
+        return pages;
+    }
+
     @Override
     public List<Page> findPagesByAttribute(String attributeName) throws SQLException{
         // Oracle String literals are stupid -> sqlInjection would be possible here...
```

Comment:
This should be documented in the javadoc for this method, maybe with links to some examples/explanations.

Reply:
#3 I've added the Javadoc in another PR.